github_deploy.admin.inc

```diff
@@ -89,7 +90,7 @@ function github_deploy_admin_settings() {
 
   $form['github_deploy']['github_deploy_admin_settings_secret'] = array(
     '#type' => 'textfield',
-    '#title' => t('Secret<span class="form-required"> *</span>'),
+    '#title' => t('Secret'),
```

Comment:
This will leave field mark as not required while its required, if you look down you will find custom validator and im using this because of states api